contracts/base/LoanTermsConsensus.sol

```diff
@@ -22,17 +27,75 @@ import "../interfaces/LoanTermsConsensusInterface.sol";
 
     @author develop@teller.finance
  */
-contract LoanTermsConsensus is LoanTermsConsensusInterface, Consensus {
-    /* Mappings */
+contract LoanTermsConsensus is LoanTermsConsensusInterface, OwnerSignersRole {
```

Comment:
- Bring logic and storage of signers over and can get rid of `_owner`. 
- Instead of using `Roles.Role` struct to keep track of the signers, we can use the `AddressArray` which means we can also get rid of `_signerCount` and just use `signers.array.length`